person_capture/gui_app.py

```
@@ -12792,30 +12832,42 @@ def _save_hdr_wic_intermediate_avif(
+                            f"HDR WIC intermediate used requested {pix_fmt}/{avif_range}",
+                            key="hdr_wic_intermediate",
+                            interval=10.0,
+                        )
+                    return True, ""
+                suffix = ":prefilter" if pre_wic_filter else ""
+                failures.append(f"{pix_fmt}{suffix}:{why}")
+                try:
+                    if os.path.exists(out_path):
+                        os.remove(out_path)
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)